### ISSUE TITLE
数据预处理论文和代码是不是不符合

### ISSUE THREAD

**Issue by xf170** [wuziqi2/CFPRec#5]
请问论文中数据集的处理中写到“We then split the trajectories of
each user in the ratio of 8:1:1 based on timestamps, where
the earliest 80% is regarded as training set; the second last
10% as validation set and the last 10% as test set.”
但是我看代码中对应的数据预处理部分并没有按照论文中描述的这样进行数据预处理。按照论文中这样处理数据的话，测试集中的用户id肯定是包含了所有用户的id，但是我看现在你这个测试集中的id只包含了几个用户，请问是这样嘛
